src/components/ContentWidgets/ImagesInRow/ImagesInRowEditor.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import "./ImagesInRow.css";
+import { Button, TextField } from "@material-ui/core";
+
+/**
+ * Diplays an editor to create/edit an ImageInRow widget
+ *
+ * Last Modified
+ * Nitesh
+ * July 13, 2019
+ */
+export const ImagesInRowEditor: React.FC<WidgetEditorProps> = ({
+	editedContent,
+	setEditedContentOnChange
+}) => {
+	interface linkCaptionPair {
+		imgLink: string;
+		imgCaption: string;
+	}
+	let pictureCaptionPairs: linkCaptionPair[] =
```

Comment:
You _must_ have all your hooks at the top of the function. Do not put **anything** before it. Move this declaration to *after* the hooks. This is just a thing with hooks.